packages/editor/src/store/test/selectors.js

```diff
@@ -1497,10 +1496,12 @@ describe( 'selectors', () => {
 
 	describe( 'isEditedPostBeingScheduled', () => {
 		it( 'should return true for posts with a future date', () => {
+			const time = Date.now() + ( 1000 * 3600 * 24 * 7 ); // sdays in the future
```

Comment:
What does "sdays" mean here?

Reply:
It means  "7 days", it's a new language invented by my broken keyboard :)